scrapy/crawler.py

```diff
@@ -35,13 +35,14 @@ def __init__(self, spidercls, settings=None):
         self.spidercls.update_settings(self.settings)
 
         d = dict(overridden_settings(self.settings))
-        logger.info("Overridden settings: %(settings)r", {'settings': d})
 
         self.signals = SignalManager(self)
         self.stats = load_object(self.settings['STATS_CLASS'])(self)
 
         handler = LogCounterHandler(self, level=self.settings.get('LOG_LEVEL'))
         logging.root.addHandler(handler)
```

Comment:
What if instead of moving `logger.info` below, we move these two lines (`handler = …`, `logging…`) above?

Reply:
:+1:

Reply:
@Gallaecio , I think some infinite loop error has occured in AppVeyor log!

Reply:
They are related indeed. It turns out the first of those two lines needs to be after `self.stats` is defined. Please ignore my suggestion.